components/aws/actions/cloudwatch-logs-put-log-event/cloudwatch-logs-put-log-event.mjs

```diff
@@ -0,0 +1,86 @@
+import aws from "../../aws.app.mjs";
+import { toSingleLineString } from "../../common/utils.mjs";
+
+export default {
+  key: "aws-cloudwatch-logs-put-log-event",
+  name: "AWS - CloudWatch Logs - Put Log Event",
+  description: toSingleLineString(`
+    Uploads a log event to the specified log stream.
+    [See docs](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-cloudwatch-logs/classes/putlogeventscommand.html)
+  `),
+  version: "0.0.2",
```

Comment:
I changed some action's keys to follow a structure, `aws-<aws_service>-<action>`.
I'm not sure if the version should go back to `0.0.1` or be bumped from the previous number.